examples/dft/24-define_xc_functional.py

```diff
@@ -30,7 +30,7 @@ def eval_xc(xc_code, rho, spin=0, relativity=0, deriv=1, verbose=None):
     rho0, dx, dy, dz = rho[:4]
     gamma = (dx**2 + dy**2 + dz**2)
     exc = .01 * rho0**2 + .02 * (gamma+.001)**.5
-    vrho = .01 * 2 * rho0
+    vrho = .01 * 2 * rho0**2
```

Comment:
It seems wrong. XC energy is `sum(weight * exc)`, so dE/drho is `.01*3*rho**2 + .02 * (gamma+.001)**.5`